ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/files/3scale_single_tenant_user.yml

```diff
@@ -0,0 +1,59 @@
+# Name of 3scale API administrator of the tenant
+- set_fact:
+    ocp4_workload_lpe_3scale_tenant_admin_id: "{{ ocp4_workload_lpe_3scale_tenant_admin_name_base }}"
+
+- set_fact:
+    ocp4_workload_lpe_3scale_org_name: "{{ocp4_workload_lpe_3scale_tenant_user_name_base}}-tenant"
+
+- set_fact:
+    ocp4_workload_lpe_3scale_tenant_gw_project: "{{ ocp4_workload_lpe_3scale_tenant_user_name_base }}-gw"
+
+
+# Get the {{ ocp4_workload_lpe_3scale_org_name}} Admin URL
+- name: Get the {{ ocp4_workload_lpe_3scale_org_name}} Admin URL
+  shell: >
+    oc get secret "{{ ocp4_workload_lpe_3scale_org_name }}-tenant-secret"  -o json -n "{{ ocp4_workload_lpe_3scale_namespace }}" | jq -r .data.adminURL | base64 -d
+  register: ocp4_workload_lpe_3scale_ocp4_workload_lpe_3scale_tenant_admin_url_reg
+  ignore_errors: True
+
+# Get the {{ ocp4_workload_lpe_3scale_org_name}} Access Token
+- name: Get the {{ ocp4_workload_lpe_3scale_org_name}} Admin URL
+  shell: >
+    oc get secret "{{ ocp4_workload_lpe_3scale_org_name }}-tenant-secret"  -o json -n "{{ ocp4_workload_lpe_3scale_namespace }}" | jq -r .data.token | base64 -d
+  register: ocp4_workload_lpe_3scale_ocp4_workload_lpe_3scale_tenant_access_token_reg
+  ignore_errors: True
+
+- set_fact:
+    ocp4_workload_lpe_3scale_tenant_admin_url: '{{ocp4_workload_lpe_3scale_ocp4_workload_lpe_3scale_tenant_admin_url_reg["stdout"]}}'
+
+- set_fact:
+    ocp4_workload_lpe_3scale_tenant_access_token: '{{ocp4_workload_lpe_3scale_ocp4_workload_lpe_3scale_tenant_access_token_reg["stdout"]}}'  
+
+- set_fact:
+        ocp4_workload_lpe_3scale_tenant_portal_endpoint: "https://{{ ocp4_workload_lpe_3scale_tenant_access_token }}@{{ ocp4_workload_lpe_3scale_org_name }}-admin.{{ ocp4_workload_lpe_3scale_ocp_apps_domain }}"
+
+- name: output tenant info
+  debug:
+    msg: "{{ item }}"
+  loop:
+    - "user.info: "
+    - "user.info: 3scale Tenant API Manager - details"
+    - "user.info: "
+    - "user.info: {{ ocp4_workload_lpe_3scale_org_name}} Admin Console : https://{{ ocp4_workload_lpe_3scale_org_name }}-admin.{{ ocp4_workload_lpe_3scale_ocp_apps_domain }}"
+    - "user.info: Admin login with credentials {{ ocp4_workload_lpe_3scale_tenant_admin_id }} / {{ ocp4_workload_lpe_3scale_tenant_admin_password }} "
+    - "user.info: "
+    - "user.info: {{ ocp4_workload_lpe_3scale_org_name}} Admin Access URL : {{ ocp4_workload_lpe_3scale_tenant_portal_endpoint }}"
+    - "user.info: "  
+  when: not silent|bool
+
+- block: 
+  - name: output gateway project info
+    debug:
+      msg: "{{ item }}"
+    loop:
+      - "user.info: "
+      - "user.info: Tenant's Gateway Project: {{ ocp4_workload_lpe_3scale_tenant_gw_project }} "
+      - "user.info: "
+      - "user.info: Tenant's OpenShift User: {{ ocp_username }}"
+    when: not silent|bool  
+  when: ocp4_workload_lpe_3scale_create_gws_with_each_tenant|bool
```

Comment:
Two comments. One name the block. Two: I prefer the when condition right after the block statement to make it very clear what/when the block does.